### PR TITLE
feat(engines): Added argument `tb_dump_img_interval` to control the i…

### PR DESCRIPTION
**Pull request by ChenglongWang**
Fix: #56 

Default interval is set to 50.

### CODE REVIEW

File: strix/models/cnn/engines/segmentation_engines.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def __init__(
             model_file_prefix=val_metric_name,
             bestmodel_n_saved=opts.save_n_best,
             tensorboard_image_kwargs=SegmentationTrainEngine.get_tensorboard_image_transform(
-                output_nc=opts.output_nc, decollate=decollate
+                output_nc=opts.output_nc, decollate=decollate, interval=1
```
Comment (urnotChloe) [inline strix/models/cnn/engines/segmentation_engines.py]: confirm interval

File: strix/models/cnn/engines/selflearning_engines.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ def __init__(
             model_file_prefix=val_metric_name,
             bestmodel_n_saved=opts.save_n_best,
             tensorboard_image_kwargs=SelflearningTrainEngine.get_tensorboard_image_transform(
-                opts.output_nc, decollate
+                opts.output_nc, decollate=decollate, interval=1
```
Comment (urnotChloe) [inline strix/models/cnn/engines/selflearning_engines.py]: confirm interval

File: strix/models/cnn/engines/segmentation_engines.py (hunk 1 above)
Comment (ChenglongWang) [reply strix/models/cnn/engines/segmentation_engines.py]: Fixed.

File: strix/models/cnn/engines/selflearning_engines.py (hunk 2 above)
Comment (ChenglongWang) [reply strix/models/cnn/engines/selflearning_engines.py]: Fixed

**Review by urnotChloe — APPROVED** [review]

**Pull request merged**